rules/ast/ast.go

```diff
@@ -732,7 +743,10 @@ func evalVectorBinop(opType BinOpType,
 	case And:
 		return lhs, true
 	case Or:
-		return lhs, true // TODO: implement OR
+		if lhs == 0 {
```

Comment:
Given:
a 0
b 1

The expression:
a or b

should return 0

Does this do that?